packages/types/environment.d.ts

```diff
@@ -1,7 +1,5 @@
 declare namespace NodeJS {
   interface ProcessEnv {
-    /** Needed to enable enterprise-only features */
-    readonly CALCOM_LICENSE_KEY: string | undefined;
```

Comment:
We should create a migration for upsetting the value to the DB first.

Reply:
Did it here: https://github.com/calcom/cal.com/pull/6574/files#diff-da0845fc4b52d249c4f05f5150127d2d9bc00151a5a1272a69afa2b8a161bad9R114